ibis/backends/base/__init__.py

```diff
@@ -456,6 +459,12 @@ class BaseBackend(abc.ABC, _FileIOHandler):
     def __init__(self, *args, **kwargs):
         self._con_args: tuple[Any] = args
         self._con_kwargs: dict[str, Any] = kwargs
+        # expression cache
+        self._cache: MutableBidirectionalMapping[
```

Comment:
Perhaps call it `_result_cache` or `_query_cache`

Reply:
Called it `_query_cache`.